cl/lib/model_helpers.py

```diff
@@ -487,3 +487,36 @@ def suppress_autotime(model, fields):
                 field.auto_now_add = _original_values[field.name][
                     "auto_now_add"
                 ]
+
+
+def parse_court_id_query(query_str: str) -> str:
```

Comment:
I don't think this belongs in the model helpers, since it's not really being used by a model and is pretty clearly search-related code. Maybe in `search_utils.py`?

Reply:
Yeah, I removed this method and its replace is now in `search_utils.py`